locust-simulations/scripts/restart_locust.sh

```
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Stop any running Locust instances
+pkill -f locust
+
+# Wait a moment for processes to terminate
+sleep 1
+
+# Start Locust with UI mode
+cd "$(dirname "$0")/.."
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)